content/en/docs/concepts/services-networking/service.md

```diff
@@ -394,36 +477,47 @@ The default is `ClusterIP`.
      load balancer will route, are automatically created.
    * [`ExternalName`](#externalname): Maps the service to the contents of the
      `externalName` field (e.g. `foo.bar.example.com`), by returning a `CNAME` record
-     with its value. No proxying of any kind is set up. This requires version 1.7 or
-     higher of `kube-dns`.
+     with its value. No proxying of any kind is set up.
+
+{{< note >}}
+
+You need CoreDNS version 1.7 or higher to use the `ExternalName` type.
```

Comment:
this swapped CoreDNS and kube-dns without changing the versioning

Reply:
Oops!

@wozniakjan would you be willing to [log an issue](https://github.com/kubernetes/website/issues/new?labels%5B%5D=kind%2Fbug&template=bug-report.md) if appropriate?

Reply:
I skipped ahead and went for a PR https://github.com/kubernetes/website/pull/18566 if that is ok

Reply:
Yeah, that's great!